# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../texinfo/wchar-mbstate-definition.patch
# Copyright (C) 2007 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- texinfo-4.11/gnulib/lib/wchar.in.h.orig	2007-07-08 10:14:45.000000000 +0200
+++ texinfo-4.11/gnulib/lib/wchar.in.h	2007-09-21 11:08:02.000000000 +0200
@@ -40,6 +40,7 @@
    Some builds of uClibc lack it.  */
 /* The include_next requires a split double-inclusion guard.  */
 #if @HAVE_WCHAR_H@
+# undef __need_mbstate_t
 # @INCLUDE_NEXT@ @NEXT_WCHAR_H@
 #endif
 
